### DISCUSSION TITLE
Group project

### DISCUSSION THREAD

**Discussion by Impavan007** [github/india#784] [General]
Hello friends is there anyone who want to work in group project i am having trouble beacause my student developer pack is not activated till now I have tried other ways for subscribition but not working so if there is anyone who want to work in group project please contact with me

**Comment by ey-mentor1** [comment]
Hello,

Can you share details  to FRT support team, they can help you in issue.
Otherwise, please create Azure free account, incase you still havent received any STD pack.

Thank you

**Reply by Impavan007** [reply]
Sir i applied several time to get student developer pack I completed self learning in ai-900 still showing in process

**Reply by ey-mentor1** [reply]
Hello,
Yes, but the STD disbursal and course progess related issues are handled by FRT support team. 
I can advice you to again check the progress of your course on Ms learn portal, check there if the course is displaying as complete.
If it's coming as completed, please send mail to FRT team with all relevant screenshots and details.
They will definitely help.

And regarding STD, I would advise to create free trial account and start working, it's free. So that, you can complete the project on time.
